### ISSUE TITLE
Insert space after snippet on character type

### ISSUE THREAD

**Issue by paraversal** [artisticat1/obsidian-latex-suite#91]
Hey! I've been crafting my personal LaTeX suite setup for a while now and it's a very well thought-out plugin, there's just one issue I keep running into:
Say you have the snippet `{trigger:"pm", replacement:"\\pm", options: "mA"}`, that works fine and if the next character you add is a number, that's okay - `$\pm4$` is valid and renders fine. But if the next character is a letter, say x, the command becomes `$\pmx$` and Obsidian (understandably) throws up an error. Now, it's possible to instead replace the snippet with `\\pm \ $0 \` which extends to `$\pm[space]$`, but that breaks math mode boundaries, since the exit dollar sign has to always be directly next to the last character. 

I assume there are other people who have run into this problem - or maybe there is even already an accepted solution.

Thanks!

**Comment by paraversal** [comment]
Ah, after browsing the closed issues for a while I think I've found the problem - like in #71, the plus-minus sign is not part of `${SYMBOL}`.

**Comment by artisticat1** [comment]
I've solved this in v1.6.8 by introducing `${SHORT_SYMBOL}`.

Replace [the default "insert space after greek letters and symbols" snippet](https://github.com/artisticat1/obsidian-latex-suite/blob/b6fbeb6aa68062d460fe2ddc49de3de3c0548b89/src/default_snippets.ts#L50) with

```typescript
    {trigger: "\\\\(${GREEK}|${SYMBOL}|${SHORT_SYMBOL})([A-Za-z])", replacement: "\\[[0]] [[1]]", options: "rmA"},
```

and that should solve the issue.
